tests/test_modules/test_perturbation_testing/test_perturbation_testing_summary.py

```diff
@@ -39,20 +41,24 @@ def test_perturbation_testing_summary(tiny_text_config):
             assert any(test.name in r.name for r in res)
 
 
-def test_without_train(tiny_text_config_no_train):
+def test_perturbation_testing_summary_one_ds(tiny_text_config_one_ds):
     mod = PerturbationTestingMergedModule(
         dataset_split_name=DatasetSplitName.all,
-        config=tiny_text_config_no_train,
+        config=tiny_text_config_one_ds,
         mod_options=ModuleOptions(pipeline_index=0),
     )
-    [res] = mod.compute_on_dataset_split()
-    assert res.eval_failure_rate >= 0.0 and res.train_failure_rate == 0.0
+    res_dict = mod.compute_on_dataset_split()[0].no_alias_dict()
+    ds_name, other_ds_name = get_tiny_text_config_one_ds_name(tiny_text_config_one_ds)
+    assert (
+        res_dict[f"{ds_name}_failure_rate"] >= 0.0
+        and res_dict[f"{other_ds_name}_failure_rate"] == 0.0
+    )
```

Comment:
Performance is not that important in tests, but just so you know, if you would like to avoid generating the whole `no_alias_dict()`, you could use the built-in Python `get_attr()` function.
```suggestion
    [res] = mod.compute_on_dataset_split()
    ds_name, other_ds_name = get_tiny_text_config_one_ds_name(tiny_text_config_one_ds)
    assert (
        get_attr(res, f"{ds_name}_failure_rate") >= 0.0
        and get_attr(res, f"{other_ds_name}_failure_rate") == 0.0
    )
```

Reply:
Oh! I'm loving it, thanks!!

Reply:
Hihiii! I'm happy you like it! :) Thanks for the feedback!